class/defaults.yml

```
@@ -1,3 +1,45 @@
 parameters:
   lieutenant:
-    namespace: syn-lieutenant
+    namespace: lieutenant
+    images:
+      operator:
+        registry: docker.io
+        repository: projectsyn/lieutenant-operator
+        version: v0.5.3
+      api:
+        registry: docker.io
+        repository: projectsyn/lieutenant-api
+        version: v0.6.0
+      steward:
+        registry: docker.io
+        repository: projectsyn/steward
+        version: v0.3.1
+    operator:
+      manifest_version: v0.5.3
```

Probably makes sense to default manifest version to image version:

```suggestion
      manifest_version: ${images:operator:version}
```